### PR TITLE
Added a gap between label and inputs #1270

### PR DESCRIPTION
**Pull request by Shaikh-Mudassir**
Added a constant spacing gap between labels and input in the following files: PasswordPanel.jsx , ProfilePanel.jsx ,TeamPanel.jsx

issue : #1720

- [ Yes] I have performed a self-review of my code.
- [ Yes] I have included the issue # in the PR.
- [ Yes] I have labelled the PR correctly.
- [ Yes] My PR is granular and targeted to one specific feature only.

### CODE REVIEW

File: Client/src/Components/TabPanels/Account/ProfilePanel.jsx [hunk 1]
```diff
@@ -223,7 +226,7 @@ const ProfilePanel = () => {
 				className="edit-profile-form"
 				noValidate
 				spellCheck="false"
-				gap={theme.spacing(20)}
+				gap={SPACING_GAP} // Applied SPACING_GAP for consistent spacing
```
Comment (marcelluscaio) [inline Client/src/Components/TabPanels/Account/ProfilePanel.jsx:229]: This is affecting the gap between groups of inputs, we also want to create spacing between label and input

File: Client/src/Components/TabPanels/Account/ProfilePanel.jsx [hunk 2]
```diff
@@ -223,7 +226,7 @@ const ProfilePanel = () => {
...
 			>
 				<Stack direction="row">
```
Comment (marcelluscaio) [inline Client/src/Components/TabPanels/Account/ProfilePanel.jsx:231]: Gap needs to be applied here

File: Client/src/Components/TabPanels/Account/ProfilePanel.jsx [hunk 3]
```diff
@@ -29,6 +29,9 @@ import Dialog from "../../Dialog";
 const ProfilePanel = () => {
 	const theme = useTheme();
 	const dispatch = useDispatch();
+	
+	// Define the constant for consistent gap
+	const SPACING_GAP = theme.spacing(12);
```
Comment (marcelluscaio) [inline Client/src/Components/TabPanels/Account/ProfilePanel.jsx:34]: This is a good thing

File: Client/src/Components/TabPanels/Account/ProfilePanel.jsx (hunk 1 above)
Comment (marcelluscaio) [inline Client/src/Components/TabPanels/Account/ProfilePanel.jsx:229]: This should be applied in all Stacks

**Review by marcelluscaio — CHANGES_REQUESTED** [review]
Nice work, and thanks for your interest in the project. I requested some changes. Also, you can remove comments from the code

File: Client/src/Components/TabPanels/Account/PasswordPanel.jsx [hunk 4]
```diff
@@ -21,6 +21,10 @@ const PasswordPanel = () => {
 	const theme = useTheme();
 	const dispatch = useDispatch();
 
+	// Define the constant for consistent gap
+	const SPACING_GAP = theme.spacing(12); // EDITED: Added a constant for gap size.
```
Comment (ajhollid) [inline Client/src/Components/TabPanels/Account/PasswordPanel.jsx:25]: These comments can be removed

**Review by ajhollid — CHANGES_REQUESTED** [review]
Looks good to me, I am in agreemtn with the changes requested by @marcelluscaio :+1: 

Once those are in I think we're good to merge here!

File: Client/src/Components/TabPanels/Account/PasswordPanel.jsx (hunk 4 above)
Comment (Shaikh-Mudassir) [reply Client/src/Components/TabPanels/Account/PasswordPanel.jsx:25]: Thank You for your feedbacks! @ajhollid  and @marcelluscaio, I'll work on the changes mentioned.

**Comment by Shaikh-Mudassir** [comment]
@ajhollid  and @marcelluscaio . I have committed the changed mentioned and also removed the comments please look into it.
Thankyou for allowing me to contribute to this project.

**Review by ajhollid — APPROVED** [review]
Looks good to me, thanks for making the requested changes

**Comment by ajhollid** [comment]
Looks good to me, as soon a @marcelluscaio approves changes I believe we can merge :+1:

**Review by marcelluscaio — APPROVED** [review]
Thank you for you contribution!

**Pull request merged**